routes/api.php

```diff
@@ -20,6 +20,6 @@
 });
 Route::get('/showposts', [PostApiController::class, 'showAll']);
 Route::get('/showpost/{post}', [PostApiController::class, 'findPost']);
-Route::post('/add', [PostApiController::class, 'add']);
 Route::put('/update/{post}', [PostApiController::class, 'update']);
+Route::post('/add', [PostApiController::class, 'add']);
 Route::delete('/delete/{post}', [PostApiController::class, 'delete']);
```

Comment:
Kindly arrange the methods: 
get, post, put, delete

Remove the /add, /update, /delete prefixes